deps/pylib.sh

```diff
@@ -1 +1,2 @@
 pip install requests_toolbelt filemagic ftputil
+pip install git+https://github.com/metabolite-atlas/metatlas
```

Comment:
Tables is a tricky one.  It is already installed in the narrative environment.  Are you running this code on dev02?